.pre-commit-config.yaml

```
@@ -114,4 +114,11 @@ repos:
         pass_filenames: false
         language: script
         types: [text]
-        files: ^(homeassistant/.+/(manifest|strings)\.json|\.coveragerc|\.strict-typing|homeassistant/.+/services\.yaml|script/hassfest/.+\.py)$
+        files: ^(homeassistant/.+/(manifest|strings)\.json|\.coveragerc|homeassistant/.+/services\.yaml|script/hassfest/.+\.py)$
```

Is there a way to exclude change to `mypy_config.py`?
```suggestion
        files: ^(homeassistant/.+/(manifest|strings)\.json|\.coveragerc|homeassistant/.+/services\.yaml|script/hassfest/((?!mypy_config).+)\.py)$
```

For me it was updating `IGNORED_MODULES` that caused me gried!

---

Good point! I've updated the regex.

---

I'm not sure if this should be adressed inside this PR, or a follow-up PR...
I noticed that the new `hassfest-metadata` ran when I updated `mypy_config`. If there are lots of hassfest triggers then it might get awkward to maintain...

---

The idea was to limit it to these three (`hassfest`, `hassfest-metadata`, `hassfest-mypy-config`). I've only created `hassfest-metadata` as it should be run on every `homeassistant/const.py` change.

> I noticed that the new `hassfest-metadata` ran when I updated `mypy_config`.

I though about if I should limit it to changes to the `metadata` script, but in the end it's fast enough that it doesn't really matter. Same btw for the `hassfest-mypy-config` script. That's also run on every change to `hassfest`, not just `mypy_config.py`.